README.md

```diff
@@ -82,15 +82,15 @@ These variables are:
 
 Now that the script is saved in your Google Drive folder we need to activate it.
 To do so click the menu "Resources" -> "Advanced Google services".  
-In the popup which will open set "Google Calendar API" to "enabled" (click the
+In the popup which will open set "Calendar API" to "enabled" (click the
 switch on its row on the right).  
-Once you have done this click on the link which says "Google API Console": you
+Once you have done this open the link which says "Google API Console" in a new tab: you
 will be taken to another page. In this page search for "Google Calendar API" and
-open it. Now click "Enable" at the top of the window and close this page.  
+open it. Now click "Enable" at the top of the window and close this page. Back where you toggled the switch ("Resources" -> "Advanced Google services") make sure "Calendar API" is still on and press "Okay".
```

Comment:
Thanks for contributing. Although @GioBonvi  will discuss/decide details of this PR with you, I'll just jump in and mention that your commit swallows the trailing spaces on this line. Trailing spaces in Markdown cause a newline without starting a new paragraph. So that would need fixing either way.